2018-07-11 13:39:44,851 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 13:39:50,372 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 13:40:05,053 ERROR: Exception on /explore-pubmed [GET] [in /home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/joseph/pygempick-flask/app/main/routes.py", line 104, in pubmed_search
    with open("app/static/to-download/pubmed_queries.txt") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'app/static/to-download/pubmed_queries.txt'
2018-07-11 13:40:06,639 ERROR: Exception on /explore-pubmed [GET] [in /home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/joseph/anaconda3/envs/pypick/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/joseph/pygempick-flask/app/main/routes.py", line 104, in pubmed_search
    with open("app/static/to-download/pubmed_queries.txt") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'app/static/to-download/pubmed_queries.txt'
2018-07-11 13:45:33,022 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 13:53:00,461 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 13:53:19,742 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 13:53:25,206 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 13:53:30,128 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:120]
2018-07-11 14:08:04,650 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 14:24:07,052 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 14:26:04,969 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 15:01:44,001 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 15:02:25,165 INFO: pyGemPick web app starting up! [in ./app/__init__.py:119]
2018-07-11 15:34:06,798 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 16:11:12,707 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 16:12:15,428 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 16:12:19,201 INFO: pyGemPick web app starting up! [in /home/joseph/pygempick-flask/app/__init__.py:119]
2018-07-11 16:13:32,192 INFO: pyGemPick web app starting up! [in ./app/__init__.py:119]
